gremlin-go/example/go.mod

```diff
@@ -22,8 +22,9 @@ go 1.20
 require github.com/apache/tinkerpop/gremlin-go/v3 v3.6.1-rc2
 
 require (
+	github.com/apache/tinkerpop/gremlin-go v0.0.0-20220530191148-29272fa563ec // indirect
```

Comment:
Not sure why this line got included, it makes me feel like something funky is happening with the go imports. I suspect it has something to do with the difference between `github.com/apache/tinkerpop/gremlin-go/v3` and `github.com/apache/tinkerpop/gremlin-go` which I've never quite understood.

Reply:
Hmm, this gives me the feeling that something isn't quite right here. Do you remember what the error was?